deployment_scripts/README.md

```diff
@@ -3,13 +3,17 @@
 
 Before running the script you must have git installed.<br>
 `Invoke-WebRequest https://raw.githubusercontent.com/guardicore/monkey/develop/deployment_scripts/deploy_windows.ps1 -OutFile deploy_windows.ps1`
+
 Then execute the resulting script with your shell. 
 First argument is an empty directory (script can create one) and second is branch you want to clone.
+
 Example usages:<br>
-`./run_script.bat` (Sets up monkey in current directory under .\infection_monkey)<br>
-`./run_script.bat "C:\test"` (Sets up monkey in C:\test)<br>
-`powershell -ExecutionPolicy ByPass -Command ". .\deploy_windows.ps1; Deploy-Windows -monkey_home C:\test"` (Same as above)<br>
-`./run_script.bat "" "master"` (Sets up master branch instead of develop in current dir)
+`.\deploy_windows.ps1` (Sets up monkey in current directory under .\infection_monkey)<br>
```

Comment:
Hmmm... Aren't additional flags like `-ExecutionPolicy ByPass` required to run ps script?